wolfcrypt/src/pkcs7.c

```diff
@@ -14878,6 +14878,74 @@ int wc_PKCS7_SetDecodeEncryptedCtx(wc_PKCS7* pkcs7, void* ctx)
 #endif /* NO_PKCS7_ENCRYPTED_DATA */
 
 
+/* Unwrap and decrypt PKCS#7/CMS EncryptedKeyPackage object, return the
+ * decoded size. */
+WOLFSSL_API int wc_PKCS7_DecodeEncryptedKeyPackage(wc_PKCS7 * pkcs7,
```

Comment:
WOLFSSL_API is only needed in the header file

Reply:
No need to change other cases in the source code for this PR. Though it should be changed to be consistent (having WOLFSSL_LOCAL/WOLFSSL_API on function declaration not implementation), I can open a PR for that.